core/src/main/scala/akka/kafka/ConsumerSettings.scala

```diff
@@ -172,6 +153,7 @@ object ConsumerSettings {
   /**
```

Comment:
regarding above `apply(system, Some(keyDeserializer), Some(valueDeserializer))`
I think we should add
```
require(keyDeserializer != null, "keyDeserializer must be defined")
require(valueDeserializer != null, "valueDeserializer must be defined")
```

Otherwise user can by mistake pass `null` and we end up with `Some(null)`